spec/jobs/topic_timer_enqueuer_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Jobs::TopicTimerEnqueuer do
+  subject { described_class.new }
+
+  fab!(:timer1) do
+    Fabricate(:topic_timer, execute_at: 1.minute.ago, created_at: 1.hour.ago, status_type: TopicTimer.types[:close])
+  end
+  fab!(:timer2) do
+    Fabricate(:topic_timer, execute_at: 1.minute.ago, created_at: 1.hour.ago, status_type: TopicTimer.types[:open])
+  end
+  fab!(:future_timer) do
+    Fabricate(:topic_timer, execute_at: 1.hours.from_now, created_at: 1.hour.ago, status_type: TopicTimer.types[:close])
+  end
+  fab!(:deleted_timer) do
+    Fabricate(:topic_timer, execute_at: 1.minute.ago, created_at: 1.hour.ago, status_type: TopicTimer.types[:close])
+  end
+
+  before do
+    deleted_timer.trash!
+  end
+
+  it "does not enqueue deleted timers" do
+    expect_not_enqueued_with(job: :close_topic, args: { topic_timer_id: deleted_timer.id })
+    subject.execute
+    expect(deleted_timer.topic.reload.closed?).not_to eq(true)
```

Comment:
```suggestion
    expect(deleted_timer.topic.reload.closed?).to eq(false)
```

I'm guessing `closed?` returns a boolean so we can just assert that it equals `false`